features/webgl2-desynchronized.yml

```diff
@@ -0,0 +1,5 @@
+name: Desynchronized WebGL2 canvas
+description: Reduces latency by desynchronizing the canvas paint cycle from the event loop.
```

Comment:
```suggestion
description: The optional `desynchronized` parameter of a canvas's `getContext()` method permits the browser to draw a WebGL2 canvas independently of the event loop. This can reduce drawing latency.
```